pull_request_template.md

```diff
@@ -1,25 +1,36 @@
 <!-- First of all, THANK YOU for your contribution. -->
 
 ## Changes Proposed:
--  
--  
+-  [ ] This pull request promotes changes to the core (units, players, creatures, game systems).
+-  [ ] This pull request promotes changes to scripts (bosses, spell scripts, creature scripts).
+-  [ ] This pull request promotes database changes.
+
+If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.
 
 ## Issues Addressed:
 <!-- If your fix has a relating issue, link it below -->
 - Closes 
 
 ## SOURCE:
 <!-- If you can, include a source that can strengthen your claim -->
+- [ ] The changes promoted by this pull request have been validated through live research (checked on live servers)
+- [ ] The changes promoted by this pull request have been validated through sniffs (remember to share them with the opensource community!)
+- [ ] The changes promoted by this pull request come from a partially or entirelly from another project (cherry-pick). **Cherry-picks must be commited using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**
```

Comment:
```suggestion
- [ ] The changes promoted by this pull request come from a partially or entirelly from another project (cherry-pick). **Cherry-picks must be commited using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**
- [ ] The changes proposed by this pull request come after checking knowledge databases (wowhead, for example)
```